imod/mf6/dis.py

```diff
@@ -88,7 +88,7 @@ class StructuredDiscretization(Package):
         "idomain": (RegridderType.OVERLAP, "mode"),
     }
 
-    _skip_mask_arrays = ["bottom", "idomain"]
+    _skip_mask_arrays = [ "bottom"]
```

Comment:
Please remove the space between [ and "
Apples also to the disv class

Reply:
done